fpdf/line_break.py

```diff
@@ -332,6 +333,25 @@ def render_pdf_text_core(self, frag_ws, current_ws):
         return ret
 
 
+class TotalPagesAliasFragment(Fragment):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.uuid = uuid4()
+
+    def get_alias_string(self):
+        return f"::alias:{self.uuid}::"
+
+    def render_pdf_text(self, *args, **kwargs):
+        self._render_args = args
+        self._render_kwargs = kwargs
```

Comment:
Shouldn't those attributes be defined in the constructor?

Reply:
I added docstrings to explain this method